src/__stories__/poster-card-story.tsx

```diff
@@ -189,8 +200,17 @@ Default.argTypes = {
         control: {type: 'select'},
     },
     background: {
-        options: ['image', 'video'],
+        options: ['image', 'video', 'color'],
+        control: {type: 'select'},
+    },
+    backgroundColor: {
+        control: {type: 'color'},
+        if: {arg: 'background', eq: 'color'},
```

Comment:
I like the solution that was implemented by vivo
https://mistica-a10w6pvfo-tuentisre.vercel.app/?path=/story/components-cards-poster-card--default

You can fill to this field some hex value (`#FFF`) or a constant (backgroundBrandSecondary) instead of color picker. (From the design side, when possible, always try to use constants and not hex values)

Reply:
hmmm... how do you know you can use a color constant name there?, that's not intuitive at all

Reply:
I've tried something different, please take a look now

Reply:
LGTM

Reply:
That's perfect. To be picky maybe we could filter tokens with "text" to avoid that appearing in the list 🌝

Reply:
that would be tricky, which colors should we exclude/include?